types/vuelidate/vuelidate-tests.ts

```diff
@@ -1,5 +1,5 @@
 import { Validation } from 'vuelidate'
-import { required, minLength, sameAs, helpers } from 'vuelidate/lib/validators'
+import { required, integer, decimal,  minLength, sameAs, helpers } from 'vuelidate/lib/validators'
```

Comment:
integer and decimal are well covered, probably will also help to cover the other typing adjustments.